src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -33,15 +33,17 @@ export default function ContentRow({
             </Typography>
           </Stack>
         </Grid>
-        <Grid item xs={12} md={9}>
-          <Box
-            sx={{
-              fontSize: 13.5,
-              overflow: "auto",
-            }}
-          >
-            {value ? <Box>{value}</Box> : <EmptyValue />}
-          </Box>
+        <Grid
+          item
+          md={9}
+          width={{xs: 1, md: 0.75}}
+          sx={{
+            fontSize: 13.5,
+            overflow: "auto",
+            marginLeft: 3,
+          }}
+        >
+          {value ? <Box>{value}</Box> : <EmptyValue />}
```

Comment:
Is this expected? https://deploy-preview-190--aptos-explorer.netlify.app/txn/151759208?network=testnet&feature=dev. The labels and values are in separate lines on non-mobile too.